src/components/Status/Status.js

```diff
@@ -0,0 +1,34 @@
+import React, { Component } from 'react';
+import { connect } from 'react-redux';
+
+class Status extends Component {
+
+  render () {
```

Comment:
As a bare minimum I would like a snapshot test of this one class with the two different messages.